projects/dsp-ui/src/lib/action/pipes/formatting/knoradate.pipe.ts

```diff
@@ -6,19 +6,36 @@ import { KnoraDate } from '@dasch-swiss/dsp-js';
 })
 export class KnoraDatePipe implements PipeTransform {
 
-    transform(date: KnoraDate, format?: string): string {
+    formattedString: string;
+
+    transform(date: KnoraDate, format?: string, precisionOptions?: 'era' | 'calendar' | 'all'): string {
         if (!(date instanceof KnoraDate)) {
             console.error('Non-KnoraDate provided. Expected a valid KnoraDate');
             return '';
         }
 
         switch (format) {
             case 'dd.MM.YYYY':
-                return `${this.leftPadding(date.day)}.${this.leftPadding(date.month)}.${date.year}`;
+                this.formattedString = `${this.leftPadding(date.day)}.${this.leftPadding(date.month)}.${date.year}`;
```

Comment:
I think when we approach [localization](https://dasch.myjetbrains.com/youtrack/issue/DSP-207), we should try to use a standard way for date formatting.

Reply:
Yeah, this pipe has gotten a bit messier than I'd like...